Framework/Source/Graphics/Model/Loaders/AssimpModelImporter.h

```diff
@@ -87,7 +88,13 @@ namespace Falcor
         void loadTextures(const aiMaterial* pAiMaterial, const std::string& folder, BasicMaterial* pMaterial, bool isObjFile, bool useSrgb);
         Material::SharedPtr createMaterial(const aiMaterial* pAiMaterial, const std::string& folder, bool isObjFile, bool useSrgb);
 
+        // Checks whether a node or its name corresponds to a used bone or node in the skeleton hierarchy
+        bool isUsedNode(const aiNode* pNode) const;
```

Comment:
Why is that in the header? It can probably be a `static` function in the CPP file.

Reply:
It checks the bone/node maps.